mmstudio/src/main/java/org/micromanager/internal/MMROIManager.java

```diff
@@ -49,7 +50,15 @@ public MMROIManager(MMStudio studio) {
     * Enquires with the UI what ROI(s) are set, and send these to the camera.
     */
    public void setROI() {
-      ImagePlus curImage = WindowManager.getCurrentImage();
+      DataViewer dv = studio_.displays().getActiveDataViewer();
+      DisplayWindow dw = null;
+      if (dv instanceof DisplayWindow) {
+         dw = (DisplayWindow) studio_.displays().getActiveDataViewer();
```

Comment:
One ends up pointing out the small things in small PRs :)
But this could be `dw = (DisplayWindow) dv;`

Reply:
Perfect!